doc/tutorial_classification.rst

```diff
@@ -6,4 +6,165 @@
+
+- How do the number of classes in the prediction sets vary according to the significance level alpha values and the conformity scores ?
+
+1. Conformal Prediction method using the softmax score of the true label
+========================================================================
+We will use MAPIE to estimate a prediction set of several classes such that the probability that the true label of a new test point is included in the prediction set is always higher than the target confidence level 1−alpha.
+
+* First we generate a dataset with train and test and the Model is fitted in the training set.
+* We set the conformal score :math:`Si = f(X_{i})_{yi}` the softmax output of the true class for each sample in the training set.
+* Then we define :math:`q` as being the :math:`(n + 1) (alpha) / n` previous quantile of :math:`S_{1}, ..., S_{n}` (this is essentially the quantile :math:`alpha`, but with a small correction). 
+* Finally, for a new test data point (where :math:`X_{n + 1}` is known but :math:`Y_{n + 1}` is not), create a prediction set :math:`T(X_{n+1}) = {y: f(X_{n+1})_{y} > q}` which includes all the classes with a sufficiently high softmax output.
+
+We use a two-dimensional dataset with three labels. The distribution of the data is a bivariate normal with diagonal covariance matrices for each label. 
+
+.. code-block:: python
+
+    import numpy as np
+    # Create training set from multivariate normal distribution
+    centers = [(0, 3.5), (-2, 0), (2, 0)]
+    # covs = [[[1, 0], [0, 1]], [[2, 0], [0, 2]], [[5, 0], [0, 1]]]
+    covs = [np.eye(2), np.eye(2)*2, np.diag([5, 1])]
+    x_min, x_max, y_min, y_max, step = -6, 8, -6, 8, 0.1
+    n_samples = 500
+    n_classes = 3
+    alpha = [0.2, 0.1, 0.05]
+    np.random.seed(42)
+    X_train = np.vstack([
+        np.random.multivariate_normal(center, cov, n_samples)
+        for center, cov in zip(centers, covs)
+    ])
+    y_train = np.hstack([np.full(n_samples, i) for i in range(n_classes)])
+
+
+    # Create test from (x, y) coordinates
+    xx, yy = np.meshgrid(
+        np.arange(x_min, x_max, step), np.arange(x_min, x_max, step)
+    )
+    X_test = np.stack([xx.ravel(), yy.ravel()], axis=1)
+
+Let's see our training data
+
+.. code-block:: python
+
+   import matplotlib.pyplot as plt
+   colors = {0: "#1f77b4", 1: "#ff7f0e", 2:  "#2ca02c", 3: "#d62728"}
+   y_train_col = list(map(colors.get, y_train))
+   fig = plt.figure()
+   plt.scatter(
+       X_train[:, 0],
+       X_train[:, 1],
+       color=y_train_col,
+       marker='o',
+       s=10,
+       edgecolor='k'
+   )
+   plt.xlabel("X")
+   plt.ylabel("Y")
+   plt.show()
+
+.. image:: images/tuto_classification_1.jpeg
+    :align: center
+
+We fit our training data with a Gaussian Naive Base estimator. And then we apply :class:`mapie.classification.MapieClassifier` with the method ``score`` to the estimator indicating that it has already been fitted.
```

Comment:
C'est plutôt l'argument `cv="prefit"` qui indique que l'estimator a déjà été fitté ?

Reply:
j'ai ajouté ça ..... it has already been fitted with `cv="prefit"`.